README.md

```diff
@@ -369,6 +369,42 @@ ZHVjdCBNYW5hZ2VtZW50MRowGAYDVQQLExFRdWFsaXR5IEFzc3VyYW5jZTEkMCIG
 A1UEAxMbbm9wcml2YXRla2V5LnZlbmFmaS5leGFtcGxlMIIBIjANBgkqhkiG9w0B
 ```
 
+## Prevent Re-issue Local
+
+### Introducing store by hash
+
+We added capability in order to store by hash, which will be the ID to read the certificate from storage.
```

Comment:
Check sentence wording here, does not make much sense.

Reply:
What do you mean? the hash will be the ID to retrieve the ID from storage

Reply:
`We added the ability to store by hash. The cert hash will be used as ID in vault storage.`

Reply:
The wording of "cert hash" sounds like the cert it self was transformed into a hash string, which is not the case. The scenario here should expose that we use (some of) certificate request data in order to create the hash which then will be the ID.

Reply:
Feel free to build upon it to give better clarity

Reply:
`We enabled capability to store certificates by hash.`